client/gutenberg/extensions/trial/block.js

```diff
@@ -100,6 +100,8 @@ class Trial extends Component {
 				className={ `status-badge status-${ status.key }` }
 				onClick={ onClick }
 				onKeyUp={ clickOnEnter }
+				role="button"
+				tabIndex="0"
```

Comment:
This isn't ideal for a11y. Without replacing the `span`, should we add `aria-label`?

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_button_role#Labeling_buttons

Reply:
Good point; it was a good chance to learn about `sprintf`  in `wp.i18n` :thumbsup:

The button will now read e.g. _"Status: New"_